plugins/inputs/solr/README.md

```diff
@@ -0,0 +1,17 @@
+# Solr input plugin
+
+The [solr](http://lucene.apache.org/solr/) plugin collects stats via the 
+[MBean Request Handler](https://cwiki.apache.org/confluence/display/solr/MBean+Request+Handler)
+
+More about [performance statistics](https://cwiki.apache.org/confluence/display/solr/Performance+Statistics+Reference)
+
+### Configuration:
+
+```
+[[inputs.solr]]
+  ## specify a list of one or more Solr servers
+  servers = ["http://localhost:8983"]
+
+  ## specify a list of one or more Solr cores
+  cores = ["main"]
```

Comment:
Thoughts on  having the plugin capture all cores by default without having to manually specify them?